roles/zookeeper/templates/myid.j2

```diff
@@ -1,11 +1,5 @@
 {% for server in zookeeper_hosts %}
-{% if server.host is defined %}
-{% if server.host == inventory_hostname %}
-{{ server.id }}
-{% endif %}
-{% else %}
-{% if server == inventory_hostname %}
 {{ loop.index }}
-{% endif %}
-{% endif %}
 {% endfor %}
+
+{{zookeeper_hosts}}
```

Comment:
Zookeeper role needs a bit work to be in usable shape. This file will eventually be modified and used.